src/multiselect.js

```diff
@@ -128,20 +128,18 @@ tvm.store = {
 
 
 tvm.createMap = async function(config) {
-  let nlmapsconf = {
-    target: config.target,
-    layer: config.layer,
-    marker: config.marker,
-    search: config.search,
-    zoom: config.zoom
-  };
-  let map = nlmaps.createMap(nlmapsconf);
+  let map = nlmaps.createMap(config);
   this.map = map;
   let parkeervakken = parkeerVakken();
   this.parkeervakken = parkeervakken;
+
+  //workaround to trigger wfsreload since map.on('load' isn't available)
+  map.on('layeradd', reloadWfs, this);
   parkeervakken.addTo(map);
+  map.off('layeradd', reloadWfs, this);
```

Comment:
Kan je niet zeggen:
```
parkeervakken.addTo(map);
reloadWfs();
```

Reply:
nee helaas niet, ik snap niet helemaal waarom, maar het heeft volgens mij te maken met dat de kaart niet aan de async functie wordt doorgegeven als je het rechtstreeks aanroept ofzo. Ik zal het hans vragen voor de zekerheid als hij er weer is.